internal/podman/client.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 var (
-	ErrPodmanNotFound  = errors.New("podman not found in your system, check requirements at http://docpage")
+	ErrPodmanNotFound  = errors.New("podman not found in your system, check requirements at https://dochub.mongodb.org/core/atlas-cli-deploy-local-reqs")
```

Comment:
[CLOUDP-200468](https://jira.mongodb.org/browse/CLOUDP-200468)

Reply:
[pro tip] Common pattern for clients that would not support automatic updates is to use url shortener with the ability to swap links at any point.

Reply:
No action needed.

Reply:
@wtrocki FYI dochub is our version of that